about.html

```diff
@@ -0,0 +1,176 @@
+        </p>
+      </section>
+
+      <section class="achievments">
+        <h2 class="ach-heading">
+          THECODECHASER SOME PAST ACHIEMENTS
+        </h2>
+        <p class="ach-details">
+          Take a look of last some achhievemnts of THECODECHASER
+        </p>
+        <div class="ach-container">
+          <div class="ach-item">
+            <img class="ach-img" alt="achievemnt image" src="images/ach-img1.jpg">
+            <div class="ach-info">
+              <h2 class="ach-info-h">2020</h2>
+              <p class="ach-info-p">Placed 300 web develoeprs</p>
+            </div>
+          </div>
+         <div class="ach-item">
+            <img class="ach-img" alt="achievemnt image" src="images/ach-img2.jpg">
+            <div class="ach-info">
+              <h2 class="ach-info-h">2019</h2>
+              <p class="ach-info-p">thecodechaser best bootcamp</p>
+            </div>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <!-- #### footer-section #### -->
+    <footer>
+
+      <!-- #### partners-section #### -->
+      <section class="partners-section" id="partners">
+        <h2 class="partners-heading">Partners</h2>
+        <hr>
+        <div class="partners-container">
+          <ul class="partners-items">
+            <li><img alt="partner brand" class="partner-img" src="images/microsoft.png"></li>
+            <li><img alt="partner brand" class="partner-img" src="images/mackafee.png"></li>
+            <li><img alt="partner brand" class="partner-img" src="images/data-expert.png"></li>
+            <li><img alt="partner brand" class="partner-img" src="images/eset.png"></li>
+            <li><img alt="partner brand" class="partner-img" src="images/tenable.png"></li>
+          </ul>
+        </div>
+      </section>
+
+      <!-- #### contact-form-section #### -->
+      <section class="contact-container" id="contact">
+        <h2 class="contact-heading">Contact Us</h2>
+        <hr class="contact-hr">
+        <p class="contact-details">Contact us for know more about TheCodeChaser, send us your quastions and quaries using this form</p>
+        <form action="https://formspree.io/f/mayazvnq" method="post" class="contact-form">
+          <input aria-label="Name" type="text" placeholder="Name*" required name="user-name" maxlength="30">
+          <input aria-label="Email" type="email" placeholder="Email*" required name="user_email">
+          <textarea cols="30" rows="10" aria-label="Message" placeholder="Type your message here*" required name="user_msg" maxlength="500"></textarea>
+        <button type="submit" class="button contact">Submit</button>
+        </form>
+      </section>
+
+      <!-- #### copyright-section #### -->
+      <section class="copyright-section">
+        <img alt="copyright logo" src="images/header-logo.png" class="copyright-logo">
+        <p class="copyright-details">2021 TheCodeChaser All Rights Reserved</p>
+      </section>
+    </footer>
```

Comment:
- [ ] `partner` section should be horizontally centered. It also should be hidden on the `about` page.
- [ ] `footer` text has 2.28:1 contrast ratio, kindly update the font color so it becomes more readable at least `4.5:1`.
- [ ] Please add more height/padding to the `footer` in on desktop screen to match the design.
- [ ] Kindly make sure that the `footer` along with the `partners` section are centered horizontally.

Reference Design
![image](https://user-images.githubusercontent.com/33767951/142336744-9306bd47-7f18-4f61-9d55-771821c37e9a.png)

![image](https://user-images.githubusercontent.com/33767951/142402648-82ff4854-3381-49f9-81d9-f49b891000a6.png)


Your Styling => '786px'
![image](https://user-images.githubusercontent.com/33767951/142399568-47053b29-54ee-41fa-8e0e-49ad3bda0d9f.png)

![image](https://user-images.githubusercontent.com/33767951/142402466-73234122-2226-4015-99dc-68d3e76af6fd.png)